api-tester/src/defaults/java/com/revenuecat/apitester/java/RedeemWebPurchaseListenerAPI.java

```diff
@@ -0,0 +1,35 @@
+package com.revenuecat.apitester.java;
+
+import androidx.annotation.OptIn;
+
+import com.revenuecat.purchases.CustomerInfo;
+import com.revenuecat.purchases.ExperimentalPreviewRevenueCatPurchasesAPI;
+import com.revenuecat.purchases.PurchasesError;
+import com.revenuecat.purchases.interfaces.RedeemWebPurchaseListener;
+
+@OptIn(markerClass = ExperimentalPreviewRevenueCatPurchasesAPI.class)
+@SuppressWarnings({"unused"})
+final class RedeemWebPurchaseListenerAPI {
+    static void checkListener(RedeemWebPurchaseListener listener,
+                              RedeemWebPurchaseListener.WebPurchaseRedeemer webPurchaseRedeemer) {
+        listener.handleWebPurchaseRedemption(webPurchaseRedeemer);
+    }
+
+    static void checkRedemptionStarter(RedeemWebPurchaseListener.WebPurchaseRedeemer webPurchaseRedeemer,
+                                       RedeemWebPurchaseListener.ResultListener resultListener) {
+        webPurchaseRedeemer.redeemWebPurchase(resultListener);
+    }
+
+    static void checkResultListener(RedeemWebPurchaseListener.ResultListener resultListener,
+                                    RedeemWebPurchaseListener.Result result) {
+        resultListener.handleResult(result);
+    }
+
+    static void checkRedeemResult(RedeemWebPurchaseListener.Result result) {
+        if (result instanceof RedeemWebPurchaseListener.Result.Success) {
```

Comment:
Maybe we could add a function `isSuccess()` to the sealed class that always returns true for `Success`? Just some syntactic sugar

Reply:
Done in https://github.com/RevenueCat/purchases-android/pull/1880/commits/810da26702bcc09a9349ba59aff76afefccbe93c